bin/node/src/commands/rpc.rs

```diff
@@ -56,6 +72,9 @@ impl RpcCommand {
             .await
             .context("Failed to bind to RPC's gRPC URL")?;
 
+        // Start system monitor.
+        std::thread::spawn(move || SystemMonitor::new(monitor_interval).run());
```

Comment:
If this is always going to run on its own thread would it make sense to include the dispatch within `SystemMonitor`